src/__screenshot_tests__/text-screenshot-test.tsx

```diff
@@ -27,3 +27,18 @@ test('Text wrapping', async () => {
     const image = await element.screenshot();
     expect(image).toMatchImageSnapshot();
 });
+
+test('Text wrapping - multiple whitespaces', async () => {
+    await openStoryPage({
+        id: 'components-text--text-wrapping',
+        device: 'DESKTOP',
+        args: {
+            text: '   a text with     spaces    everywhere   Pneumonoultramicroscopicsilicovolcanoconiosis test   ',
```

Comment:
Sadly, I can't test an example where I use \n, because storybook has some weird behavior that removes all the args from the URL if I put the \ character

Reply:
Good idea. Done :)